thoth/storages/graph/postgres.py

```diff
@@ -3357,13 +3383,42 @@ def sync_solver_result(self, document: dict) -> None:
                         index_url,
                         solver_info,
                     )
+
+                    package_metadata, _ = PythonPackageMetadata.get_or_create(
+                        self._session,
+                        author=importlib_metadata.pop("Author", None),
+                        author_email=importlib_metadata.pop("Author-email", None),
+                        classifier=importlib_metadata.pop("Classifier", None),
+                        download_url=importlib_metadata.pop("Download-URL", None),
+                        home_page=importlib_metadata.pop("Home-page", None),
+                        keywords=importlib_metadata.pop("Keywords", None),
+                        # package licence
+                        license=importlib_metadata.pop("License", None),
+                        maintainer=importlib_metadata.pop('Maintainer', None),
+                        maintainer_email=importlib_metadata.pop('Maintainer-email', None),
+                        metadata_version=importlib_metadata.pop("Metadata-Version", None),
+                        # package name
+                        name=importlib_metadata.pop("Name", None),
+                        platform=importlib_metadata.pop("Platform", None),
+                        requires_dist=importlib_metadata.pop("Requires-Dist", None),
+                        summary=importlib_metadata.pop("Summary", None),
+                        # package version
+                        version=importlib_metadata.pop("Version", None),
+                    )
+
+                    if importlib_metadata:
+                        raise PythonPackageMetadataAttributeMissing(
+                            f"{importlib_metadata.keys()} keys not in PythonPackageMetadata."
```

Comment:
Please be more descriptive with error so later anyone can immediately say what's wrong, so something like:

```
No related columns for {list(importlib_metadata.keys())!r} found in PythonPackageMetadata table, cannot sync the whole solver result
```